scripts/package_firefox_extension.sh

```diff
@@ -40,9 +40,9 @@ if [ "${TAG}zz" == "zz" ]; then
     else
             echo "Extension not signed by Mozilla. It might be because this commit id has already been signed : let's look for it in a previous nightly build"
             FOUND=0
-            for FILE in $(ssh -i ../scripts/travisci_builder_id_key ci@download.kiwix.org "find /data/download/nightly -name \"kiwix-firefox-signed-extension-$VERSION.xpi\""); do
+            for FILE in $(ssh -o StrictHostKeyChecking=no -i ./scripts/ssh_key ci@download.kiwix.org "find /data/download/nightly -name \"kiwix-firefox-signed-extension-$VERSION.xpi\""); do
```

Comment:
It's probably ../scripts/ssh_key instead of ./scripts/ssh_key

Reply:
I did wonder. Please feel free to make any changes you notice, as you have a much better grasp of how these scripts are working (and of bash syntax) than I do.

Reply:
I'll make these changes now.